Sources/TokamakCore/Views/Layout/HStack.swift

```diff
@@ -27,7 +27,8 @@ public let defaultStackSpacing: CGFloat = 8
 ///     }
 public struct HStack<Content>: View where Content: View {
   public let alignment: VerticalAlignment
-  let spacing: CGFloat
+  @_spi(TokamakCore)
+  public let spacing: CGFloat?
```

Comment:
formatting readability nit: at a glance this looks like `@_spi(TokamakCore)` is applied to both properties without the newline
```suggestion
  @_spi(TokamakCore)
  public let spacing: CGFloat?

```